repository/s.json

```diff
@@ -5555,17 +5555,6 @@
 				}
 			]
 		},
-		{
-			"name": "Swift",
```

Comment:
Please revert this removal of the Swift package, since we now intend to leave it where it is and just add your package as a new and separate one.

Reply:
Done

Reply:
@yaroslavyaroslav you removed the previous_names, but the removal of Swift from the index is still in the PR.

Reply:
@braver yep, my bad, now everything should be fine